File: support/SDKSampleJava/app/src/main/java/com/gyant/sdksamplejava/DisplayGyantChatActivity.java
package com.gyant.sdksamplejava;

import android.os.Bundle;

import com.gyant.chat_sdk.GyantChat;
import com.gyant.chat_sdk.GyantChatActivity;

public class DisplayGyantChatActivity extends GyantChatActivity {

    @Override
    public void onCreate(Bundle savedInstanceState) {
        GyantChat.getInstance()
                .clientId("client_id")
                .start();

        super.onCreate(savedInstanceState);
    }
}

File: support/SDKSampleKotlin/app/src/main/java/com/gyant/sdksamplekotlin/DisplayGyantChatActivity.kt
package com.gyant.sdksamplekotlin

import android.os.Bundle

import com.gyant.chat_sdk.GyantChat
import com.gyant.chat_sdk.GyantChatActivity

class DisplayGyantChatActivity : GyantChatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        GyantChat.getInstance()
            .clientId("client_id")
            .isDev(true)
            .start()
        super.onCreate(savedInstanceState)
    }
}
